src/encoded/static/components/browse/components/SelectedFilesController.js

```diff
@@ -155,9 +160,13 @@ export class SelectedFilesController extends React.PureComponent {
             logger.error("Supplied accessionTriple is not a string or array of strings/arrays:", accessionTriple);
             throw new Error("Supplied accessionTriple is not a string or array of strings/arrays:", accessionTriple);
         }
-        const newlyAddedFileItems = [];
+        //it is initialized inside of setState, since setState is called twice
+        //in strict mode and newlyAddedFileItems would have duplicate items
+        //more info: https://github.com/facebook/react/issues/12856#issuecomment-390206425
```

Comment:
Thanks for clarifying this. I remember Alex had explained this at one point, but I couldn't remember the exact reason.

Reply:
Yeah, I've also lost 2 hours determining why it is not working. Dan Abramov had a good explanation in the link.